build/common.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <Import Project="dependencies.props" />
-  <Import Project="..\version.props" />
+  <Import Project="..\version.xml" />
```

Comment:
That's the plan. I think this helps make it clearer that the file can be used as raw XML, not just an MSBuild file.